packages/arb-rpc-node/cmd/arb-node/arb-node.go

```diff
@@ -196,20 +196,22 @@ func startup() error {
 		sequencerFeed = make(chan broadcaster.BroadcastFeedMessage, 1)
 		for _, url := range config.Feed.Input.URLs {
 			broadcastClient := broadcastclient.NewBroadcastClient(url, nil, config.Feed.Input.Timeout)
-			for {
-				err = broadcastClient.ConnectWithChannel(ctx, sequencerFeed)
-				if err == nil {
-					break
+			go (func() {
```

Comment:
Worth refactoring into a `waitForFeedConnection` function that can be shared between this and the relay?